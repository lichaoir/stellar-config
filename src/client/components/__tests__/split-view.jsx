const React = require('react');
const SplitView = require('../split-view.jsx');
const renderer = require('react-test-renderer');

describe('component split-view', () => {
  test('wraps two children', () => {
    const component = renderer.create(
      <SplitView>
        <div>Child 1</div>
        <div>Child 2</div>
      </SplitView>
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
