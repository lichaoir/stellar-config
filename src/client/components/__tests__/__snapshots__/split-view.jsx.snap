// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component split-view wraps two children 1`] = `
<div>
  <div
    className="pane is-half is-pulled-left"
  >
    <div>
      Child 1
    </div>
  </div>
  <div
    className="pane is-half is-pulled-right"
  >
    <div>
      Child 2
    </div>
  </div>
</div>
`;
